Mail/Components/IndeterminateProgressView.swift

```diff
@@ -0,0 +1,51 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import SwiftUI
+
+struct IndeterminateProgressView: View {
+    @State private var animation = 1.0
+    let indeterminate: Bool
+    let progress: Double
+    var body: some View {
```

Comment:
```suggestion
    @State private var animation = 1.0
    
    let indeterminate: Bool
    let progress: Double
    
    var body: some View {
```
Small request: add new lines to make the view more readable